### PR TITLE
chore: add portainer service to docker-compose

### PR DESCRIPTION
**Pull request by tmberthold**
Note: Do not merge yet, since AppStore support not yet on Dataspace Connector main.

### CODE REVIEW

File: dataspace-connector/full/README.md [hunk 1]
```diff
@@ -18,15 +18,17 @@ To get and install your own IDS identity certificate, please read the provided [
...
 * PostgreSQL DB
 
 ### Components & Versions
 This deployment example builds on the following compatible versions:
 
 | Component | Version |
 |:----------|:--------|
-| Dataspace Connector | 6.1.0 |
-| Dataspace Connector UI | 8.3.2 |
+| Dataspace Connector | 6.1.3 |
```
Comment (brianjahnke) [inline dataspace-connector/full/README.md:29]: The version of the DSC will change to 6.2.0.

File: dataspace-connector/full/README.md [hunk 2]
```diff
@@ -18,15 +18,17 @@ To get and install your own IDS identity certificate, please read the provided [
...
+| Dataspace Connector UI | 8.4.ß |
```
Comment (brianjahnke) [inline dataspace-connector/full/README.md:30]: "8.4.ß" <- is that last "number" correct?

Comment (tmberthold) [reply dataspace-connector/full/README.md:30]: no

File: dataspace-connector/full/README.md (hunk 1 above)
Comment (ob-silva) [reply dataspace-connector/full/README.md]: I changed the version to 6.2.0

Comment (tmberthold) [reply dataspace-connector/full/README.md]: needs to be changed in docker-compose too

Comment (ob-silva) [reply dataspace-connector/full/README.md]: I changed it in the docker-compose to 6.2.0

**Review by brianjahnke — APPROVED** [review]

**Pull request merged**